src/styles/scss/abstracts/_mixins.scss

```diff
@@ -0,0 +1,61 @@
+@mixin header-shadow {
+  box-shadow: 0px 2px 10px rgba(22, 22, 22, 0.1);
+}
+
+@mixin common-shadow {
+  box-shadow: 0px 0px 10px rgba(22, 22, 22, 0.1);
+}
+
+@mixin extra-small-text-regular {
+  font-family: $type-family;
+  font-weight: $type-weight;
+  font-style: normal;
+  font-size: 14px;
+  line-height: 18px;
+  color: $text;
```

Comment:
usually  we want to have possibly to use another color in text. 
Let's add possibility to pass custom color to mixin

Reply:
add this customisation 

example:

@mixin extra-small-text-regular($color: $custom-black) {
  @include custom-text;
  font-weight: $type-weight;
  font-size: 14px;
  line-height: 18px;
  @if $color {
    color: $color;
  }
}